package.json

```diff
@@ -17,15 +17,18 @@
   },
   "dependencies": {
     "brace": "^0.8.0",
+    "classnames": "^2.2.5",
```

Comment:
Also classnames is a good package in general so we can make use of it in other areas.